Zend/tests/fibers/gh10496.phpt

```diff
@@ -0,0 +1,33 @@
+--FILE--
```

Comment:
You swapped the FILE & TEST section titles.

Reply:
Oops :-) fixed